### PR TITLE
Updated capsnet example

### PR DESCRIPTION
**Pull request by ThomasDelteil**
## Description ##

- small fixes to use mx.test_utils.download instead of urllib
- use mxboard instead of tensorboard

### CODE REVIEW

**Comment by roywei** [comment]
@Thanks for the contribution!
@mxnet-label-bot [pr-awaiting-review,Example]

File: example/capsnet/README.md [hunk 1]
```diff
@@ -1,66 +1,66 @@
...
+```
+Full arguments  
+```
+python capsulenet.py --batch_size 100 --devices gpu0,gpu1 --num_epoch 100 --lr 0.001 --num_routing 3 --model_prefix capsnet
+```  
+
+* * *
+## **Prerequisities**
+
+MXNet version above (0.11.0)
```
Comment (srochel) [inline example/capsnet/README.md]: do we have tests MXBoard is working with 0.11.0?
```suggestion
MXNet version above (0.11.0)  
```
MXNet version above (1.2.0)

**Comment by ankkhedia** [comment]
@safrooze @thomelane @sergeykolychev Could you please take a look into this PR?

**Comment by anirudhacharya** [comment]
@ThomasDelteil this example does not have a gluon implementation. Should the PR be held off until the example gets updated?

**Comment by ThomasDelteil** [comment]
@anirudhacharya it is fine to keep a symbolic example when they meet the bar in terms of quality. Ideally we'll rewrite it in Gluon later, but we can keep it as it is for now since there are no Gluon implementation. I have simply fixed that one to use mxboard instead of tensorboard

**Comment by anirudhacharya** [comment]
LGTM, for the defined scope of the PR. 

@indhub @sandeep-krishnamurthy can you please merge this.

**Comment by ThomasDelteil** [comment]
@indhub @nswamy @gigasquid @eric-haibin-lin @szha @sandeep-krishnamurthy @indhub can you please merge? Thanks!

**Pull request merged**